### PR TITLE
Signers count metric.

### PR DESCRIPTION
**Pull request by Frozen**
Signers count metric for prometheus.

### CODE REVIEW

**Review by ONECasey — APPROVED** [review]

File: consensus/leader.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func (consensus *Consensus) onPrepare(recvMsg *FBFTMessage) {
 	signerCount := consensus.decider.SignersCount(quorum.Prepare)
 	//// Read - End
 
-	consensus.UpdateLeaderMetrics(float64(signerCount), float64(consensus.getBlockNum()))
```
Comment (sophoah) [inline consensus/leader.go:134]: for the leader lets keep UpdateLeaderMetrics

File: consensus/metrics.go [hunk 2]
```diff
@@ -149,6 +149,12 @@ func (consensus *Consensus) UpdateLeaderMetrics(numCommits float64, blockNum flo
 	consensusCounterVec.With(prometheus.Labels{"consensus": "num_commits"}).Add(numCommits)
 	consensusGaugeVec.With(prometheus.Labels{"consensus": "num_commits"}).Set(numCommits)
 }
+
+func (consensus *Consensus) SignersCountMetrics(signersCount float64, blockNum float64) {
+	consensusCounterVec.With(prometheus.Labels{"consensus": "signers_count"}).Add(signersCount)
+	consensusGaugeVec.With(prometheus.Labels{"consensus": "signers_count"}).Set(signersCount)
+}
```
Comment (sophoah) [inline consensus/metrics.go:156]: i don't think we need SignersCountMetrics since we already have `UpdateValidatorMetrics`

**Comment by Frozen** [comment]
docker run -d -p 9091:9091 prom/pushgateway
if start node with flags "--prometheus.push=true" "--prometheus.pushgateway=http://127.0.0.1:9091"
you can get data at http://127.0.0.1
image

**Comment by sophoah** [comment]
i don't think we need that PR anymore, i will close it for now since we now have the signing power broadcasted in the network

**Pull request closed without merging**